test/parse-args.js

```diff
@@ -43,6 +43,26 @@ describe('parse-args', () => {
       const argv = buildYargs().parse(['node', 'c8', 'my-app'])
       argv.lines.should.be.equal(95)
     })
+    it('should resolve be able to resolve config files with .js extensions', () => {
+      const argv = buildYargs().parse(['node', 'c8', 'my-app', '--config', require.resolve('./fixtures/config/.nycrc.js')])
+      argv.lines.should.be.equal(95)
+    })
+    it('should resolve be able to resolve config files with .yml extensions', () => {
+      const argv = buildYargs().parse(['node', 'c8', 'my-app', '--config', require.resolve('./fixtures/config/.nycrc.yml')])
+      argv.lines.should.be.equal(99)
+    })
+    it('should resolve be able to resolve config files with .yaml extensions', () => {
+      const argv = buildYargs().parse(['node', 'c8', 'my-app', '--config', require.resolve('./fixtures/config/.nycrc.yaml')])
+      argv.lines.should.be.equal(98)
+    })
+    it('should resolve be able to resolve config files with .cjs extensions', () => {
+      const argv = buildYargs().parse(['node', 'c8', 'my-app', '--config', require.resolve('./fixtures/config/.nycrc.cjs')])
+      argv.lines.should.be.equal(94)
+    })
+    it('should resolve be able to resolve config files with .json extensions', () => {
+      const argv = buildYargs().parse(['node', 'c8', 'my-app', '--config', require.resolve('./fixtures/config/.nycrc.json')])
+      argv.lines.should.be.equal(96)
+    })
```

Comment:
New unit tests for additional configuration files can be [found here](https://github.com/bcoe/c8/pull/436/commits/ccbad85fec2c0fbc1450ceeb9d7c20d49beda480#diff-aa8ddbaaf379948f2e332b5b4ef3a1c71387520b3a2eca9d5f6aeba377e00bbf)